src/slack-integration.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+set -e
+
+throw() {
+  echo "Error: $*"
+  exit 1
+}
+
+[ -z "$SLACK_CI_CHANNEL_RELEASE" ] && throw "Environment variable $SLACK_CI_CHANNEL_RELEASE is not set."
```

Comment:
Use single quotes here, otherwise `$SLACK_CI_CHANNEL_RELEASE` will be interpolated.

```suggestion
[ -z "$SLACK_CI_CHANNEL_RELEASE" ] && throw 'Environment variable $SLACK_CI_CHANNEL_RELEASE is not set.'
```